android/src/toga_android/widgets/selection.py

```diff
@@ -87,3 +166,10 @@ def rehint(self):
         self.native.measure(View.MeasureSpec.UNSPECIFIED, View.MeasureSpec.UNSPECIFIED)
         self.interface.intrinsic.width = at_least(self.native.getMeasuredWidth())
         self.interface.intrinsic.height = self.native.getMeasuredHeight()
+
+    def set_font(self, font):
+        self._font_impl = font._impl
+        tv = self.native.getSelectedView()
+        if tv:
+            self.adapter.apply_font(tv)
+        self.interface.refresh()
```

Comment:
This line should be unnecessary, as the style system should automatically do a refresh whenever any style changes.

Reply:
I removed the refresh